aws-ecs-service/alb.tf

```diff
@@ -30,7 +30,7 @@ resource "aws_lb_target_group" "service" {
 resource "aws_lb" "service" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"